Integrations/integration-Okta.yml

```diff
@@ -2,7 +2,6 @@ commonfields:
   id: okta
   version: -1
 name: okta
-system: true
 display: okta
```

Comment:
example for what ?
Place this file in your res directory, 
change the assetid in the content-descriptor file
and restart the server
okta should be available, and treated as system (no drill in)